easybuild/easyconfigs/c/ColabFold/ColabFold-1.5.5-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,187 @@
+            'filename': '%(name)s-colabfold-%(version)s.tar.gz'}],
+        'testinstall': True,
+        'checksums': [
+            {'AlphaFold-colabfold-2.3.6.tar.gz': 'a12868525e568091acc5d8e15e9fc389125199786b750243b567ad1c4f89779b'},
+            {'AlphaFold-2.3.2_data-dep-paths-shebang-UniRef30.patch':
+             '58cd0ce4094afe76909649abe68034c4fbdb500967f5c818f49b530356dc012b'},
+            {'AlphaFold-2.0.0_n-cpu.patch': 'dfda4dd5f9aba19fe2b6eb9a0ec583d12dcefdfee8ab8803fc57ad48d582db04'},
+            {'AlphaFold-2.0.1_setup_rm_tfcpu.patch':
+             '1a2e4e843bd9a4d15ee39e6c37cc63ba281311cc7a0a5610f0e43b52ef93faac'},
+            {'AlphaFold-2.3.2_BioPythonPDBData.patch':
+             'e4483a525ae5c4dc5a5f633bed8cf5337c329e64b603ab7b684a9d18cd26a22f'},
+        ],
+    }),
+    ('colabfold', version, {
+        'patches': ['%(name)s-%(version)s_relax_requirements.patch'],
+        'preinstallopts': """sed -i 's/^\\(python =.*\\)<=3.10"$/,<=%(pyshortver)s"/;""" +
+                          """s/flow-cpu/flow/' pyproject.toml &&""",
+        'source_urls': ['https://github.com/%(github_account)s/%(name)s/archive'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'use_pip_extras': 'alphafold',
+        'checksums': [
+            {'colabfold-1.5.5.tar.gz': '7fb0ad263c24ec00d46f2dff1611989234bc1351d71653dd06006f2c88f83dcb'},
+            {'colabfold-1.5.5_relax_requirements.patch':
+             'bcbcd7f403fea2e98ea04568d6ef358c72ad4beab762e701903473abf1883d03'},
+        ],
+    }),
+]
+
+_alphafold_buildir = '%%(builddir)s/AlphaFold/alphafold-%s' % _sokrypton_commit
+postinstallcmds = [
+    "mkdir -p %(installdir)s/bin",
+    "cp %s/run_alphafold.py %%(installdir)s/bin/run_alphafold.py" % _alphafold_buildir,
+    "chmod a+x %(installdir)s/bin/run_alphafold*.py",
+    "cd %(installdir)s/bin && ln -s run_alphafold.py alphafold",
+    "cp -a %s/scripts %%(installdir)s/" % _alphafold_buildir,
+    # Copy notebooks to installation directory
+    "mkdir %(installdir)s/notebooks",
+    "cp AlphaFold2.ipynb ESMFold.ipynb RoseTTAFold.ipynb %(installdir)s/notebooks/",
+]
+
+sanity_check_paths = {
+    'files': [
+        'bin/run_alphafold.py', 'bin/colabfold_batch', 'bin/colabfold_search', 'bin/colabfold_split_msas',
+        'lib/python%(pyshortver)s/site-packages/alphafold/common/stereo_chemical_props.txt'
+    ],
+    'dirs': ['lib/python%(pyshortver)s/site-packages', 'notebooks'],
+}
+
+sanity_check_commands = [
+    "pdbfixer --help",
+    "run_alphafold.py --help 2>&1 | grep 'Full AlphaFold protein structure prediction script'",
+    "colabfold_batch --help",
+]
+
+sanity_pip_check = True
+
+modextravars = {
+    'TF_FORCE_UNIFIED_MEMORY': '1',
+    'XLA_PYTHON_CLIENT_MEM_FRACTION': '3',
+    # 'ALPHAFOLD_DATA_DIR': '/path/to/AlphaFold_DBs/2.3.2',  # please adapt
+    'ALPHAFOLD_DATA_DIR': '/scratch/AlphaFold_DBs/2.3.2',  # please adapt,
```

Comment:
I would remove this line, and just not set `$ALPHAFOLD_DATA_DIR` by default, we can't use any sensible default value here anyway...

```suggestion
```